nupic/research/frameworks/pytorch/imagenet/mixins/rezero_weights.py

```diff
@@ -44,8 +44,8 @@ def setup_experiment(self, config):
                               float(nonzero_params_sparse2) / params_sparse)
 
     @classmethod
-    def create_model(cls, config, device):
-        model = super().create_model(config, device)
+    def create_model(cls, config):
```

Comment:
refactoring this method signature will break any code that rely on the device being passed to `create_model`.
Just run `git grep 'create_model'` in this and any other repo that depends on `nupic.research` to get a list of files that need to be updated as well.

Reply:
Fixed